<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>645</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>761</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="pageLayoutVertical">
     <item>
      <widget class="QGroupBox" name="pageBox">
       <property name="styleSheet">
        <string notr="true">#pageBox {
background: rgb(255, 255, 255);
}
</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>130</y>
          <width>761</width>
          <height>291</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="bodyPageLayoutVertical">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>50</number>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLineEdit" name="nome_box">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">#nome_box{ 
	border-width: 1px;
    border-style: solid;
    border-radius: 4px;
}</string>
             </property>
             <property name="placeholderText">
              <string>*Nome</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="cognome_box">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">#cognome_box {
	border-width: 1px;
    border-style: solid;
    border-radius: 4px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>*Cognome</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLineEdit" name="username_box">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">#username_box {
	border-width: 1px;
    border-style: solid;
    border-radius: 4px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>*Numero di celllulare o e-mail</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_15">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLineEdit" name="nPassword_box">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">#nPassword_box {
	border-width: 1px;
    border-style: solid;
    border-radius: 4px;
}</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>*Nuova Password</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>100</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="leftMargin">
              <number>10</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_17">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Giorno</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="giorno">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#giorno {
	border-width: 1px;
	border-style: solid;
	border-color: black;
	border-radius: 5px;
}</string>
                 </property>
                 <property name="specialValueText">
                  <string>21</string>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>31</number>
                 </property>
                 <property name="singleStep">
                  <number>1</number>
                 </property>
                 <property name="stepType">
                  <enum>QAbstractSpinBox::DefaultStepType</enum>
                 </property>
                 <property name="value">
                  <number>1</number>
                 </property>
                 <property name="displayIntegerBase">
                  <number>10</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_16">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Mese</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="mese">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#mese {
	border-width: 1px;
	border-color:black;
	border-style: solid;
	border-radius: 4px;
}</string>
                 </property>
                 <property name="specialValueText">
                  <string>1</string>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>12</number>
                 </property>
                 <property name="stepType">
                  <enum>QAbstractSpinBox::DefaultStepType</enum>
                 </property>
                 <property name="value">
                  <number>1</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_18">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                   <weight>50</weight>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Anno</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="anno">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#anno {
	border-width: 1px;
	border-color:black;
	border-style: solid;
	border-radius: 4px;
}</string>
                 </property>
                 <property name="specialValueText">
                  <string>1995</string>
                 </property>
                 <property name="minimum">
                  <number>1900</number>
                 </property>
                 <property name="maximum">
                  <number>2020</number>
                 </property>
                 <property name="stepType">
                  <enum>QAbstractSpinBox::DefaultStepType</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_20">
               <item>
                <widget class="QRadioButton" name="sessoD">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>16</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Donna</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_22">
               <item>
                <widget class="QRadioButton" name="sessoM">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>16</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Uomo</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QPushButton" name="iscriviti_button">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">#iscriviti_button {
	color: white;
	border-width: 1px;
    border-style: solid;
    border-radius: 4px;
	border-color: black;
	background: rgb(43, 140, 7);
}
</string>
             </property>
             <property name="text">
              <string>Iscriviti</string>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>761</width>
          <height>131</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="headerPageLayoutVertical_5">
         <item>
          <widget class="QGroupBox" name="loginBox">
           <property name="styleSheet">
            <string notr="true">#loginBox {
background-color:#3b5998;
}
</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="verticalLayoutWidget_14">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>90</y>
              <width>761</width>
              <height>89</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayoutPassDim">
             <property name="leftMargin">
              <number>35</number>
             </property>
             <item alignment="Qt::AlignTop">
              <widget class="QPushButton" name="RecuperoPassword">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">#RecuperoPassword {
	color:#9cb4d8;
}</string>
               </property>
               <property name="text">
                <string>Non riccordi più come accedere all'account?</string>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>761</width>
              <height>91</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>30</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                <widget class="QLabel" name="label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#label{
	color: white;

}</string>
                 </property>
                 <property name="text">
                  <string>Email o telefono</string>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignHCenter|Qt::AlignTop">
                <widget class="QLineEdit" name="emailLogin_box">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>16</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#emailLogin_box {
	border-width: 1px;
    border-style: solid;
    border-radius: 4px;
}</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_12">
               <property name="spacing">
                <number>0</number>
               </property>
               <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                <widget class="QLabel" name="label_2">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#label_2{
	color: white;
}</string>
                 </property>
                 <property name="text">
                  <string>Password</string>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignHCenter|Qt::AlignTop">
                <widget class="QLineEdit" name="passwordLogin_box">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>16</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#passwordLogin_box {
	border-width: 1px;
    border-style: solid;
    border-radius: 4px;
}</string>
                 </property>
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>20</number>
               </property>
               <property name="rightMargin">
                <number>25</number>
               </property>
               <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                <widget class="QPushButton" name="pushButton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>18</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#pushButton {
	border-width: 2px;
    border-style: solid;
    border-radius: 4px;
	border-color: #29487d;	
	background-color: #4267b2;
 	color: white;
}</string>
                 </property>
                 <property name="text">
                  <string>Accedi</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
